### PR TITLE
Added Blog and blog post

### LINKED ISSUE #11: Prototyp a demo blog entry
The blog entry should have at least the following content:

- title
- description
- audio section
- video section
- image section

You are welcome to use dummy content for demonstration purposes.

### LINKED ISSUE #13: Create first data structure and first demo entries in CMS
In order to showcase the advantages of using a CMS, some sample data related to our project would support the front end development.

### LINKED ISSUE #14: Connect CMS API with front end
Provide the demo content of the CMS by means of an restful API as well as consume and render it in the front end prototype.

### PR DESCRIPTION
**Pull request by larsrickert**
- Frontend: Added single blog post page, closes #11 
- Frontend: Added blog page that shows all blogs, filterbar needs to be implemented in separate PR
- Backend: Added blog post content type, closes #13 
- Connected frontend and backend, closes #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,76 +12,42 @@ replace me with information about how we work and how someone could support this
...
 
-### `store`
+- Frontend will be available on: http://localhost:3000
+- Backend will be available on: http://localhost:1337
 
-This directory contains your Vuex store files. Creating a file in this directory automatically activates Vuex.
+### Step 3: Configure Backend (database)
 
-More information about the usage of this directory in [the documentation](https://nuxtjs.org/docs/2.x/directory-structure/store).
+1. Login to strapi
```
Comment (philippabele) [inline README.md:50]: How do I login to strapi at localhost:1337 ?

File: frontend/pages/blog/index.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+    const { axios } = useAxios()
+
+    axios
+      .get<IBlogPost[]>('blog-posts')
+      .then(({ data }) => {
+        blogPosts.value = data
+      })
+      .catch((e) => {
+        console.error(e)
```
Comment (philippabele) [inline frontend/pages/blog/index.vue:78]: Why will errors only be printed to console? Does the user get no feedback, if there is an error?

File: frontend/pages/blog/index.vue [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      <b-overlay
+        :show="isLoading"
+        rounded
+        opacity="0.6"
+        spinner-small
+        spinner-variant="primary"
+        class="col-lg-9 mx-3 mx-md-0"
+      >
+        <template v-if="!isLoading">
+          <p v-if="!blogPosts.length" class="text-muted">
```
Comment (philippabele) [inline frontend/pages/blog/index.vue:26]: There should be a distinction between no blog posts from the API and an error thrown during the GET request.

**Review by philippabele — CHANGES_REQUESTED** [review]
The page looks good and the connection to the backend works. For the purpose of malfunction of the backend, I killed my local db and the strapi container. By running the frontend in standalone mode, I discovered only minor adjustments which need to be fixed.

**Comment by larsrickert** [comment]
> The page looks good and the connection to the backend works. For the purpose of malfunction of the backend, I killed my local db and the strapi container. By running the frontend in standalone mode, I discovered only minor adjustments which need to be fixed.

Thanks. 

> The page looks good and the connection to the backend works. For the purpose of malfunction of the backend, I killed my local db and the strapi container. By running the frontend in standalone mode, I discovered only minor adjustments which need to be fixed.

Thanks. I fixed your change requests.

**Pull request merged**